autowelcome.py

```diff
@@ -0,0 +1,42 @@
+from brutal.core.plugin import BotPlugin, cmd, event
+
+
+class Welcome(BotPlugin):
+    def setup(self, *args, **kwargs):
+        self.welcomes = self.open_storage('autowelcomes')
+
+    @cmd
+    def welcome(self, event):
+        """Create an autowelcome message for yourself."""
+        args = event.args
+
+        if len(args) < 1:
+            return "No new autowelcome message specified."
+
+        nick = event.meta['nick']
+        welcome = ' '.join(args[0:])
+        self.welcomes[nick] = welcome
+
+        return "Your new autowelcome is: {0}".format(welcome)
+
+    @cmd
+    def my_welcome(self, event):
```

Comment:
@JakubNvk yeah, maybe the `if len(args) < 1:` part on line 35 could return your welcome message.